sale_order_type/tests/test_sale_order_type.py

```diff
@@ -4,6 +4,7 @@
 # License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
 
 import odoo.tests.common as common
+from odoo.tests import Form
```

Comment:
```suggestion
from datetime import timedelta

from odoo import fields
import odoo.tests.common as common
from odoo.tests import Form
```
Related to previous suggestion it will be necessary to.